submissions/sayopaul/Week1/fizz.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Nice work @sayopaul, I actually prefer the longer version because it is more readable than the shorter version. Always try to write human readable code.

Reply:
Alright . Thank you very much Sir for the feedback . I wouldn't forget that